### PR TITLE
Add REFRESH SCHEMA to ALTER SOURCE

### LINKED ISSUE #1936: Document: feat: alter the schema registry of table with connector
This issue tracks the documentation update needed for the merged PR #15025.

Source PR URL: https://github.com/risingwavelabs/risingwave/pull/15025
Source PR Merged At: 2024-03-08T05:22:07Z

### LINKED ISSUE #1969: Document: feat: refresh the schema of sources with schema registry
This issue tracks the documentation update needed for the merged PR #15541.

Source PR URL: https://github.com/risingwavelabs/risingwave/pull/15541
Source PR Merged At: 2024-03-18T08:24:42Z

### PR DESCRIPTION
**Pull request by ShanlanLi**
## Info

- **For reviewer:**
- [Modify schema page](https://pr-2008.d2fbku9n2b6wde.amplifyapp.com/docs/dev/modify-schemas/)
- [Alter source refresh schema page](https://pr-2008.d2fbku9n2b6wde.amplifyapp.com/docs/dev/sql-alter-source/#refresh-schema)
- [Alter table page refresh schema page](https://pr-2008.d2fbku9n2b6wde.amplifyapp.com/docs/dev/sql-alter-table/#refresh-schema)

- **Notes**

  - [ Include any supplementary context or references here. ]

- **Related code PR**

  - https://github.com/risingwavelabs/risingwave/pull/15541/files
  - https://github.com/risingwavelabs/risingwave/pull/15025

- **Related doc issue**
  
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/1969#event-12150518701
 Resolves https://github.com/risingwavelabs/risingwave-docs/issues/1936#event-12150428262

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

**Comment by emile-00** [comment]
Should also update this page: https://docs.risingwave.com/docs/current/modify-schemas/

File: docs/sql/commands/sql-alter-table.md [hunk 1]
```diff
@@ -193,3 +193,22 @@ ALTER TABLE table_name
...
 ```
+
+### `REFRESH SCHEMA`
+
+```sql title=Syntax
+ALTER TABLE table_name
+    REFRESH SCHEMA;
+```
+
+This clause alters the schema registry of the able with connectors.
```
Comment (emile-00) [inline docs/sql/commands/sql-alter-table.md]:
```suggestion
This command alters the schema registry of a table created with connectors.
```

Comment (emile-00) [reply docs/sql/commands/sql-alter-table.md]: A bit wordier, but I think the "which was created" part makes things more clear. .

**Review by emile-00 — APPROVED** [review]

**Comment by ShanlanLi** [comment]
@fuyufjh Could you please help review this PR? The owner of its code PR, the intern Rossil seemed to have left RisingWave. I saw you are the reviewer of that code PR, so ask for your help.

**Review by fuyufjh — APPROVED** [review]
LGTM

**Pull request merged**